examples/siro_sandbox/README.md

```diff
@@ -43,7 +45,7 @@ python examples/siro_sandbox/sandbox_app.py \
     * Left-click again to drop the object anywhere in the scene.
```

Comment:
How about update this controls section for the free camera: explain free-camera controls (arrow keys to move) and GUI-controlled humanoid controls (left-click to grab, etc.).